tck/src/main/java/org/eclipse/microprofile/health/tck/OnlySuccessfulReadinessTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICES file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.microprofile.health.tck;
+
+import java.io.StringReader;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.eclipse.microprofile.health.tck.deployment.FailedHealth;
+import org.eclipse.microprofile.health.tck.deployment.SuccessfulHealth;
+import org.eclipse.microprofile.health.tck.deployment.SuccessfulLiveness;
+import org.eclipse.microprofile.health.tck.deployment.SuccessfulReadiness;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.shrinkwrap.api.Archive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.eclipse.microprofile.health.tck.DeploymentUtils.createWarFileWithClasses;
+import static org.eclipse.microprofile.health.tck.JsonUtils.asJsonObject;
+
+/**
+ * @author Antoine Sabot-Durand
+ */
+public class OnlySuccessfulReadinessTest extends SimpleHttp {
```

Comment:
I would delete this test but add a test in the previous test to test both liveness and readiness.

Reply:
+1 maybe also add just health to the previous test if it's not covered somewhere else